figures/views.py

```diff
@@ -899,8 +899,10 @@ class SiteViewSet(StaffUserOnDefaultSiteAuthMixin, viewsets.ReadOnlyModelViewSet
     Access is restricted to global (Django instance) staff
     """
     model = Site
-    queryset = Site.objects.all()
```

Comment:
You should probably just be able to set `queryset = figures.sites.get_sites()` and not need the `get_queryset` method. I'd need to look through the DRF code to see if there is any significant difference

Reply:
If `get_sites` returns an already [evaluated list (and memory cached)](https://stackoverflow.com/a/40618173/161278) there could be a problem of having the list of sites stale. 

If it did have a stale sites, it means the only way to refresh the list is to restart the workers.

While this issue may not happen, it would be a rabbit hole to discover it.

Therefore I'm erring on the safe side of using a method since it's out of the control of Figures.

Reply:
Fair enough